scripts/gen_regex.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import unicodedata
+from ftfy import chardata
+import pathlib
+from pkg_resources import resource_filename
+
+
+DATA_PATH = pathlib.Path(resource_filename('wordfreq', 'data'))
+
+
+def _emoji_char_class():
+    """
+    Build a regex for emoji substitution.  We create a regex character set
+    (like "[a-cv-z]") matching characters we consider emoji.
+    """
+    emoji_file = DATA_PATH / 'emoji.txt'
```

Comment:
This variable is unused, it seems?  (The inline style seems nicer, if you want to make it consistent between these functions.)